test/index.js

```diff
@@ -182,5 +231,30 @@ describe('lozad', () => {
       assert.equal('true', image.dataset.loaded)
       assert.equal(image.getAttribute('src'), image.dataset.src)
     })
+
+    // It('should run loaded function after loading an element', () => {
```

Comment:
this seems unwanted, better remove the commented code

Reply:
👍